pandas/tests/apply/test_frame_apply.py

```diff
@@ -1551,3 +1551,11 @@ def foo(x):
     df = DataFrame({"a": [1, 2, 3]})
     with tm.assert_produces_warning(UserWarning, match="Hello, World!"):
         df.agg([foo])
+
+
+def test_apply_return_type():
```

Comment:
there are other tests for return_type can you co-locate

Reply:
Have relocated the file to `pandas/tests/frame/methods/test_astype.py`